libs/libc/Makefile

```diff
@@ -194,9 +196,9 @@ distclean:: clean
 	$(Q) $(MAKE) -C kbin distclean
 	$(Q) $(MAKE) -C zoneinfo distclean BIN=$(BIN)
 	$(call DELFILE, exec_symtab.c)
-	$(call DELFILE, bin/Make.dep)
-	$(call DELFILE, kbin/Make.dep)
+	$(call DELFILE, bin$(DELIM)Make.dep)
+	$(call DELFILE, kbin$(DELIM)Make.dep)
 	$(call DELFILE, .depend)
 
--include bin/Make.dep
--include kbin/Make.dep
+-include bin$(DELIM)Make.dep
```

Comment:
What is an issue with using `$(DELIM)`?

Reply:
use `/` here result failure of make distclean

Reply:
Sorry for the confusion. It was more a general question not related to this line directly. I see many places that replace `DELIM` with `/`, so wondering why that is done?

Reply:
> Sorry for the confusion. It was more a general question not related to this line directly. I see many places that replace `DELIM` with `/`, so wondering why that is done?

#1088 remove the line `DELIM := $(strip /)`
`# REVISIT: Backslash causes problems in $(COBJS) target
DELIM := $(strip /)
BINDIR ?= bin

AOBJS = $(patsubst %.S, $(BINDIR)$(DELIM)%$(OBJEXT), $(ASRCS))
COBJS = $(patsubst %.c, $(BINDIR)$(DELIM)%$(OBJEXT), $(CSRCS))`
in Windows native build give an error:
`makefile:132: *** target mode do not include“%”。 stop。`

ditto mm/Makefile